docs/DeveloperGuide.md

```diff
@@ -1004,36 +1004,94 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
+   2. Navigate to the folder using your command prompt.
+   3. Launch the jar file using the ```java -jar ta2.jar```. 
+   4. Expected: Shows the GUI with a set of sample contacts. No assignments are displayed under the Assignments panel. The window size may not be optimum. The image below is the window you will see upon starting TA<sup>2</sup>.
+<br/><br />
+![Sample data in TA<sup>2</sup>](images/ManualTestingSampleData.PNG) <br /><br />
+2. Saving window preferences
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Resize the window to an optimum size. Move the window to a different location. Close the window. 
 
-1. Saving window preferences
-
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
+### Adding a person
+1. Adding a person while all persons are being shown.
+   1. Prerequisites: List all persons using the `list` command. Ensure there is no person named "Stephen Fallon" in the list before proceeding. 
+   2. Test case: `add n/Stephen Fallon m/CS2100 e/E1337123@u.nus.edu t/L21`<br>
+   Expected: Appends added contact to your SoC contact list. Details of the added contact shown in the status message. The image below shows the result of this command on the contact list if you started with the 6 people from the sample data.<br /><br /> ![Manual Testing for Adding Person](images/ManualTestingAddingPerson.PNG) <br /><br />
+   
+   3. Test case: Repeat `add n/Stephen Fallon m/CS2100 e/E1337123@u.nus.edu t/L21` again <br>
+   Expected: No person is added. Error details shown in the status message because the peron, "Stephen Fallon", already exists in the list.<br/><br />
+
+2. Adding a person while only some persons are being shown.
+   1. Prerequisites: Ensure there is at least two persons in your contact list. Display a subset of persons using `find n/Stephen Fallon` command assuming Stephen Fallon is one of the persons in the contact list. Feel free to use the `find` command for any other persons in your contact list instead. Check that there is no one named "Hawking Einstein" in your contact list.
```

Comment:
“there are”

Reply:
maybe i change to "there are no persons named" for clarity.